system/mp/test/mptest.cpp

```diff
@@ -1466,11 +1482,16 @@ bool createNodeList(IArrayOf<INode> &nodes, const char* hostfile, int my_port, r
         if ((max_ranks > 0) && ((i - 1) >= max_ranks))
             break;
 
-        int srtn = sscanf(line, "%s", hoststr);
-        if (srtn == 1 && line[0] != '#') {
-            INode* newNode = createINode(hoststr, my_port);
-            nodes.append(*newNode);
-            i++;
+        int srtn = sscanf(line, "%255s", hoststr);
```

Comment:
not new perhaps, but feels like the:
```
if (line[0] != '#')
```

should come 1st before the sscanf

Reply:
Yes, changed.